Uplift/Supporting/ClientStrings.swift

```diff
@@ -49,6 +49,11 @@ struct ClientStrings {
         static let noFavoritesText = "FIND YOUR\nFAVORITE NOW."
         static let vcTitleLabel = "Favorites"
     }
+    
+    struct Facilities {
+        static let closed = "Closed"
+        static let open = "Open"
```

Comment:
is `open` a keyword?

Reply:
I believe we also already have "Open" and "Closed" elsewhere in `ClientStrings` (but in a different struct) -- to avoid repeated code I think we should create a new struct for strings used throughout the app and move the commonly used strings there